src/components/events/event/PendingUsers.vue

```diff
@@ -1,35 +1,94 @@
 <template>
-  <ul class="allowed-users list-group">
-    <li v-for="(user, index) in pendingUsers" :key="index" class="list-group-item">
-      <div class="d-flex w-100 justify-content-between mb-3">
-        <h5 class="mb-1">{{ user.publicName }}</h5>
-        <span class="badge badge-success badge-pill status-indicator" v-if="user.online">online</span>
-        <span class="badge badge-danger badge-pill status-indicator" v-else>offline</span>
-      </div>
+  <div class="pending-users">
+    <ul class="allowed-users list-group">
+      <li v-for="(user, index) in eventUsersByEvent" :key="index" class="list-group-item">
+        <div class="d-flex w-100 justify-content-between mb-3">
+          <span  class="user-information mb-1">
+            <h5 class="mb-0">{{ user.publicName }}</h5>
+            <small>{{ user.username }}</small>
+          </span>
+          <span class="badge badge-success badge-pill status-indicator" v-if="user.online">online</span>
+          <span class="badge badge-danger badge-pill status-indicator" v-else>offline</span>
+        </div>
 
-      <button class="btn btn-success mr-2">{{ localize('view.event.user.verifyAs') }} {{ localize('view.event.user.member') }}</button>
-      <button class="btn btn-info mr-2">{{ localize('view.event.user.verifyAs') }} {{ localize('view.event.user.visitor') }}</button>
-      <button class="btn btn-danger mr-2">{{ localize('view.event.user.block') }}</button>
-    </li>
-  </ul>
+        <button class="btn btn-success mr-2" v-on:click="verifyWithVoting(user.id)">{{ localize('view.event.user.verifyAs') }} {{ localize('view.event.user.member') }}</button>
+        <button class="btn btn-info mr-2" v-on:click="verifyAsGuest(user.id)">{{ localize('view.event.user.verifyAs') }} {{ localize('view.event.user.visitor') }}</button>
+        <button class="btn btn-danger mr-2">{{ localize('view.event.user.block') }}</button>
+      </li>
+    </ul>
+  </div>
 </template>
 
 <script>
 
 import { localize } from '@/helper/localization-helper'
+import { EVENTUSERS_BY_EVENT } from '@/graphql/queries'
+import { updateEventUserVerified } from '@/graphql/mutations'
```

Comment:
Please export the gql as constant like the line above.

Reply:
mutations are defined as functions and not as const

Reply:
nope

Reply:
The old code was also "wrong".